### ISSUE TITLE
Selective sync folders to show rather than all or none

### ISSUE THREAD

**Issue by weiwang11** [nextcloud/desktop#2157]
Users should be able to selectively display synced folders rather than having one option to enable or disable all synced folders in explorer's navigation pane. Same applies for other systems as well.

**Comment by jospoortvliet** [comment]
Can you explain a bit better what this is? You can choose what folders to sync, but you talk about displaying in Explorer - is that an Explorer issue perhaps? Or are you talking about the Virtual Drive?

**Comment by er-vin** [comment]
My understanding was that the Explorer navigation pane integration (windows specific) is all or nothing. If you got ten sync connections they all appear in that Explorer sidebar or none of them appear. I think what @weiwang11 is after is a way to have some of them appear but not all of them.

Is that correct?

And indeed screenshots could help.

**Comment by weiwang11** [comment]
Yes what @er-vin said was correct and I added a screenshot regarding the issue.

**Comment by ghalib5000** [comment]
any news on this? this should help remove all the folders showing or none at all, what i want to do is to just show the main nextcloud folder and not other auxilary folders i've synced with my system, the option to just have em all or none or them doesn't make this possible atm.

**Comment by mr-manuel** [comment]
This feature is also requested as a part of https://github.com/nextcloud/desktop/issues/5045

**Comment by Rello** [comment]
The client has improved a lot since the ticket was opened.
If it is still relevant, please create a fresh ticket.
Thank you for your support
